src/coreclr/src/inc/corjitflags.h

```diff
@@ -40,7 +42,6 @@ class CORJIT_FLAGS
         CORJIT_FLAG_TARGET_P4               = 9,
         CORJIT_FLAG_USE_FCOMI               = 10, // Generated code may use fcomi(p) instruction
         CORJIT_FLAG_USE_CMOV                = 11, // Generated code may use cmov instruction
-        CORJIT_FLAG_USE_SSE2                = 12, // Generated code may use SSE-2 instructions
```

Comment:
Just a note for a potential future PR, if we are removing this, it might be worth revisiting some of the others as well.

`CMOVcc` was in the Pentium Pro (1995) while `SSE` came later in the Pentium III (1999).
`SSE2` came in the Pentium 4 which I presume is the `TARGET_P4` we have.

Reply:
This corjit flag isn't really removed. We now have a separate InstructionSet_SSE2 for it, along with all of the rest of the instruction set stuff.

We could probably remove the CMOV and FCOMI stuff as it isn't really used for anything, and the TARGET_P4 is only tied to an optimization that I bet we never hit. (TARGET_P4 generates code that runs slower on any other core than a P4, but runs subtly better on a P4). I'm not really trying to make a bigger change than this though, so if you want to adjust that, please do it in another PR.